packages/layout/src/ChartPanel.ts

```diff
@@ -515,7 +515,11 @@ ChartPanel.prototype.publish("titleVisible", true, "boolean");
 ChartPanel.prototype.publish("titleOverlay", false, "boolean");
 ChartPanel.prototype.publishProxy("titleIcon", "_titleBar");
 ChartPanel.prototype.publishProxy("titleIconFont", "_titleBar");
+ChartPanel.prototype.publishProxy("titleIconFontColor", "_titleBar");
 ChartPanel.prototype.publishProxy("titleFont", "_titleBar");
+ChartPanel.prototype.publishProxy("titleFontBold", "_titleBar");
+ChartPanel.prototype.publishProxy("titleFontColor", "_titleBar");
+ChartPanel.prototype.publishProxy("titleBackgroundColor", "_titleBar", "backgroundColor");
```

Comment:
I should probably have just named this `titleBackgroundColor` in `TitleBar` like the others. It just seemed odd that "title" preceeded so many publish properties in `TitleBar` (I think that was my own doing).